src/common-utils/main.sh

```diff
@@ -196,7 +196,10 @@ install_redhat_packages() {
             grep \
             which \
             man-db \
-            strace"
+            strace \
+            tree \
+            bzip2 \
```

Comment:
Makes sense to install them as this is widely used.

Reply:
Hi, 

The tools we noticed were missing was tree and bzip2.

bzip2 was found threw some build scripts, so it could be changed.

The rest were found throw comparison rather than a specific need for them.